src/test/java/seedu/letsgethired/model/InternTrackerTest.java

```diff
@@ -125,4 +126,14 @@ public ObservableList<InternApplication> getApplicationList() {
         }
     }
 
+    @Test
+    public void same_InternTracker_is_equal() {
+        assertEquals(internTracker, internTracker);
+    }
+
+    @Test
+    public void differentObjectIsNotEqual() {
+        Object obj = new Object();
+        assertFalse(internTracker.equals(obj));
+    }
```

Comment:
Perhaps we can follow the convention of `unitBeingTested_descriptionOfTestInputs_expectedOutcome` for the test names?

Alternatively, consider following the convention in some of the other test classes!
![image](https://github.com/AY2324S1-CS2103T-W17-2/tp/assets/49238630/c4df0cad-3511-44d2-ba2a-00fdc5bff86e)